1 Missing Fr localisation

Fr-Dae opened the pull request:
https://github.com/Fr-Dae/laserfence-Dae/issues/1
https://github.com/randomflyingtaco/laserfence/issues/2 
    - Add Fr loc
    - Cleanup injo.js

Fr-Dae committed:
fr Loc

    - Add Fr loc
    - Cleanup injo.js

Fr-Dae committed:
fck me, fix numb

ix numb

randomflyingtaco commented on info.json:
```diff
@@ -1,17 +1,20 @@
+	"version": "2.0.4",
+    "factorio_version": "2.0",
 	"dependencies": [
-		"base >= 2.0",
```
Need to restrict these versions to 2.0 because they use are not backwards compatible with 1.1

randomflyingtaco reviewed (changes requested):
Please discard changes to info.json and changelog.txt I've committed versions of those that fit with the styling of the rest of the documents. I'd like to keep the English name as "Laser Fence" but feel free to spice up the French name however makes since for that community
I also added 2 missing mod-setting-description strings laserfence-enabled and laserfence-disabled to the English localisation since they were previously hard-coded. Add those to you base.cfg and I'll merge this request

Fr-Dae committed:
base >= 2.0"

base >= 2.0"

Fr-Dae committed:
Fix conflicts

Fix conflicts

Fr-Dae committed:
Fix changelogs conflicts

Fix changelogs conflicts

Fr-Dae commented:
Hi, thanks for your review,
here are the details of the changes made, 
- I've added a unicode emoji icon, which will be displayed in game (highlighting the mod in the list and with colour in this sad menu) as well as on the discord update list (a way of catching the eye).
- I've added the word ‘c&c’ to the title, in both French and English, to make it easier to search by linked content.
- in french we generally leave the title in english for a quick search, many french speakers understand french, english, spanish/german. it is therefore common to use a mixture of languages (especially english).

- for the info.json, I've made a visual modification to improve readability and a form of standardisation in 3 ‘blocks’ to make modification quicker. (I use geany and gitkraken)

- unless I'm mistaken, you can force merge, the conflict won't be a problem.

Fr-Dae commented:
also, and although I don't agree with it all, 
- I have applied the ‘standard’ format used by the factorio community, which can be found on the official wiki.
- I personally use the iso 8601 standard (yyyy-mm-dd) which has the advantage of getting everyone to agree.
so your changelog will handle the date display correctly.

randomflyingtaco committed:
Update info.json

Fr-Dae commented on info.json:
```diff
@@ -1,20 +1,19 @@
-	"author": "randomflyingtaco, Sacredanarchyc",
-    "contact": "https://github.com/randomflyingtaco/laserfence",
-    "homepage": "https://discord.gg/freXKz5",
+	"author": "randomflyingtaco and Sacredanarchy",
```
if you note it like that , it's like one uniqu personne

Fr-Dae commented on info.json:
```diff
@@ -1,20 +1,19 @@
-    "homepage": "https://discord.gg/freXKz5",
+	"author": "randomflyingtaco and Sacredanarchy",
+	"contact": "https://github.com/randomflyingtaco/laserfence",
+	"homepage": "https://discord.gg/freXKz5",
```
if you use tab and not space you will have some problem, don't ask me why, i have no idea why it's make mage crash

randomflyingtaco replied:
Tabs are fine to indent in info.json maybe you are thinking of changelog.txt which require 2 spaces for indenting?

Fr-Dae commented on info.json:
```diff
@@ -1,20 +1,19 @@
 	"name": "laserfence",
-	"title": "⚡ C&C Laserfence",
-	"description": "⚡Use Laser Fence technology that was developed during the Tiberium Wars to fend off biters.",
+	"title": "Laser Fence",
```
the unicode emojie look so cool in mod menu ;_; you don't like it ?

randomflyingtaco replied:
I like the name as simply "Laser Fence"
I added Command and Conquer to the description to make it more searchable and you can title the French version however you want

Fr-Dae reviewed (commented):
.

randomflyingtaco committed:
Update info.json

randomflyingtaco committed:
Merge branch 'main' into 1-missing-fr-localisation
